### PR TITLE
fix api controller auto loading

### PR DESCRIPTION
**Pull request by buddhaCode**

### CODE REVIEW

File: source/developers-guide/rest-api/plugin-api-extension/index.md [hunk 1]
```diff
@@ -75,7 +75,7 @@ This is our plugin bootstrap which subscribes to two events. For the one thing i
 `Enlight_Controller_Dispatcher_ControllerPath_Api` event to register the API controller and for 
 the other thing it uses the `Enlight_Controller_Front_StartDispatch` event to register an additional namespace for 
 our plugin. We do that, because then the API manager class can load our resource with `\Shopware\Components\Api\Manager::getResource('Banner')`.
-Since `Shopware 5.2.17` the last part is not necessary anymore. You can simply use the services.xml file to register new resources to Shopware.
+Since `Shopware 5.2.17`, both parts is not necessary anymore. You can simply use the services.xml file to register new resources to Shopware. The API controller will be found via autoloading.
```
Comment (uehler) [inline source/developers-guide/rest-api/plugin-api-extension/index.md]:
```
both parts is not necessary anymore
```

are not

File: source/developers-guide/rest-api/plugin-api-extension/index.md [hunk 2]
```diff
@@ -75,7 +75,7 @@ This is our plugin bootstrap which subscribes to two events. For the one thing i
 `Enlight_Controller_Dispatcher_ControllerPath_Api` event to register the API controller and for 
 the other thing it uses the `Enlight_Controller_Front_StartDispatch` event to register an additional namespace for 
 our plugin. We do that, because then the API manager class can load our resource with `\Shopware\Components\Api\Manager::getResource('Banner')`.
-Since `Shopware 5.2.17` the last part is not necessary anymore. You can simply use the services.xml file to register new resources to Shopware.
+Since `Shopware 5.2.17`, both parts are not necessary anymore. You can simply use the services.xml file to register new resources to Shopware. The API controller will be found via autoloading.
```
Comment (mitelg) [inline source/developers-guide/rest-api/plugin-api-extension/index.md]: are you sure the autoloading of controllers happens with `5.2.17`, too?

I summon you @shyim :smile:

Comment (buddhaCode) [reply source/developers-guide/rest-api/plugin-api-extension/index.md]: I think so. 5.2.7 is the smallest tag for this commit https://github.com/shopware/shopware/commit/e181c8cba776118dcf92bbda51aca1e7478b1959#diff-96c7e4e98b5a46f19925922a7fd4f4c0

Comment (mitelg) [inline source/developers-guide/rest-api/plugin-api-extension/index.md]: I would like to add that, it is necessary, that the directory is important, must be correct, or something like this

Comment (shyim) [reply source/developers-guide/rest-api/plugin-api-extension/index.md]: Looks good for me. But I would add example where the file has to been

Comment (buddhaCode) [reply source/developers-guide/rest-api/plugin-api-extension/index.md]: Good hint. A have added a reference to the controllers chapter.

**Comment by mitelg** [comment]
thanks for your contribution @buddhaCode 👍🎉💙

**Pull request merged**